pkg/bootstrap/boostrapkubeconfig.go

```diff
@@ -181,6 +187,26 @@ func GetKubeAPIServerAddress(ctx context.Context, client client.Client,
 	return "", err
 }
 
+func GetKubeconfigClusterName(ctx context.Context, client client.Client) (string, error) {
+	defaultCluster := "default-cluster"
+	if !helpers.DeployOnOCP {
```

Comment:
We should add a comment here. We will miss the function if hub is non-ocp, but we are trying to support that.

Reply:
PTAL